### PR TITLE
Add new Progress events to help diagnose RefreshPorts & Tunnel Service Requests

### PR DESCRIPTION
**Pull request by jramsay**
Add new Progress events to help diagnose RefreshPorts & Tunnel Service Requests

### CODE REVIEW

**Comment by jramsay** [comment]
Please see this PR for proposed usage:
https://github.com/microsoft/dev-tunnels/pull/377

**Review by IlyaBiryukov — APPROVED** [review]

File: src/cs/Ssh/Progress.cs [hunk 1]
```diff
@@ -24,4 +24,18 @@ public enum Progress
 	CompletedKeyExchange,
 	StartingSessionAuthentication,
 	CompletedSessionAuthentication,
+
+	// Port forwarding
+	StartingRefreshPorts,
+	CompletedRefreshPorts,
+
+	// Tunnel service requests
```
Comment (jasongin) [inline src/cs/Ssh/Progress.cs:32]: Please avoid adding anything specific to the tunnel service in this repo. I try to keep this library generally useful for any applications working with SSH. While these constants won't cause any problems, I'd rather not set a precedent.

Comment (jramsay) [reply src/cs/Ssh/Progress.cs:32]: Sounds good. I'll likely abandon these changes and have a completely different TunnelProgress enumerartion for the TunnelService specific progress events.

**Comment by jramsay** [comment]
abandoning as I'm adding these new events to a separate enumeration in tunnels-sdk

**Pull request closed without merging**